db/migrate/20201013171632_remove_contact_types_from_case_contacts.rb

```diff
@@ -0,0 +1,6 @@
+class RemoveContactTypesFromCaseContacts < ActiveRecord::Migration[6.0]
+  def change
+    # case_contacts.contact_types has been deprecated in favor of case_contact.case_contact_contact_type
+    remove_column :case_contacts, :contact_types, :string, array: true, default: []
```

Comment:
This will break the case contacts contact types backfill rake task that we struggled with for staging, and which isn't in prod yet. We don't have a test for it yet (just fixed it in rails console last night) so, uh, nothing will fail here. 
We can work around that by deploying current-staging to prod before deploying this code to staging (and thence to prod). I am happy to merge this, but we (I) have to remember this when deploying. And the rake task will need to be modified for the sake of new contributors getting set up locally.